Registry/Registry.EM_COMMON

```diff
@@ -1307,6 +1307,33 @@ state    real  RQGSHTEN        ikj      misc        1         -      r        "R
 state    real  RQCNSHTEN       ikj      misc        1         -      r        "RQCNSHTEN"             "Q_CN TENDENCY DUE TO SHALLOW CUMULUS SCHEME"     "# kg-1 s-1"
 state    real  RQINSHTEN       ikj      misc        1         -      r        "RQINSHTEN"             "Q_IN TENDENCY DUE TO SHALLOW CUMULUS SCHEME"     "# kg-1 s-1"
 
+# Deng Shallow Convection Parameterization
+state    real  RDCASHTEN       ikj      misc        1         -      r        "RDCASHTEN"             "DCA TENDENCY DUE TO DENG SHALLOW CUMULUS SCHEME"    "Pa s-1"
+state    real  RQCDCSHTEN      ikj      misc        1         -      r        "RQCDCSHTEN"            "DCQC TENDENCY DUE TO DENG SHALLOW CUMULUS SCHEME"   "kg kg-1 s-1"
+state    real  CLDAREAA        ikj      dyn_em      1         -      r        "CLDAREAA"              "CLOUD FRACTION FROM DENG SCHEME" ""
+state    real  CLDAREAB        ikj      dyn_em      1         -      r        "CLDAREAB"              "CLOUD FRACTION FROM DENG SCHEME" ""
+state    real  CA_RAD          ikj      dyn_em      1         -      r        "CA_RAD"                "CLOUD WATER CONTENT FROM DENG SCHEME FOR RADIATION" "kg kg-1"
+state    real  CW_RAD          ikj      dyn_em      1         -      r        "CW_RAD"                "CLOUD WATER CONTENT FROM DENG SCHEME FOR RADIATION" "kg kg-1"
+state    real  CLDLIQA         ikj      dyn_em      1         -      r        "CLDLIQA"               "CLOUD WATER CONTENT FROM DENG SCHEME" "kg kg-1"
+state    real  CLDLIQB         ikj      dyn_em      1         -      r        "CLDLIQB"               "CLOUD WATER CONTENT FROM DENG SCHEME" "kg kg-1"
+state    real  CLDDPTHB         ij      misc        1         -      r        "CLDDPTHB"              "CLOUD DEPTH FROM TAU-1"                "m"
+state    real  CLDTOPB          ij      misc        1         -      r        "CLDTOPB"               "CLOUD TOP FROM TAU-1"                "m"
+state    real  PBLMAX           ij      misc        1         -      r        "PBLMAX"                "MAXIMUM VALUE OF PBL DEPTH SINCE THE START OF THE MODEL RUN"                 "m"
+state    real  WUB             ikj      misc        1         -      r        "WUB"                   "UPDRAFT VERTICAL VELOCITY"                 "m s-1"
+state    real  RAINSHVB         ij      misc        1         -      r        "RAINSHVB"              "TIME-STEP SHCU PRECIPITATION FROM TAU-1"       "m s-1"
```

Comment:
@dudhia Is there a reason to use a different time-step RAINSH or RAINSHV? If so, should the rain be considered in the time-step total that goes into LSM?

Reply:
Not clear rainshvb is used for anything.
rainshv is used for the LSM

Reply:
may want to remove it later